### ISSUE TITLE
Vagrant 2.1.3 plugin startup error

### ISSUE THREAD

**Issue by btburton42** [hashicorp/vagrant#10193]
### Vagrant version
Vagrant: 2.1.3
VirtualBox: 5.2.18 r124319 (Qt5.6.3)

### Host operating system
Mac OS: v10.12.6 (16G1510)

### Guest operating system
CentOS-6.4-i386-v20130309

### Vagrantfile
we are using a ruby common module and requiring that into each Vagrant file, but the offending line seems to be just requiring the vagrant artifactory plugin at the top of our module
```ruby
require 'yaml'
require 'artifactory'

module VagrantConfig
     {...lots of unrelated stuff...}
    def get_artifactory_client
      unless @artifactory
        Artifactory.configure do |art_config|
          art_config.endpoint = @config['repo_url']
        end
        @artifactory = Artifactory::Client.new
      end
    end
     {...more unrelated stuff...}
end
```

### Debug output
https://gist.github.com/btburton42/9df2e6b3649e4dd44fb1ecf04e55b430

### Expected behavior
Vagrant should have started

### Actual behavior
Vagrant errored out

**Comment by pouyana** [comment]
Have the same error in the Vagrant `2.1.3` and `2.1.4` with `vagrant-openstack-provider`:

Vagrant version
===========

Vagrant: 2.1.3/2.1.4
Driver: OpenStack

Host
====

Mac OS: v10.12.6 (16G1510)


Guest operating system
==================

Ubuntu-Bionic

My Vagrant file:

```ruby
require 'vagrant-openstack-provider'
... unrelated ...
```

My debug output:
https://gist.github.com/pouyana/027f466882cd6babb66150a97e344926

Vagrant 2.1.2 works as intended.

**Comment by chrisroberts** [comment]
Fixed by #10199
